### PR TITLE
Convert to runtime: ruby.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,12 +41,7 @@ Then, run the Rails web server:
...
 
-    $ gcloud preview app deploy app.yaml --set-default
-
-This application uses a custom `Dockerfile` based on the [`google/ruby`][] image.
-
-The `Dockerfile` also runs `bundle install --without development:test` so that only
-production gems will be installed in production.
+    $ gcloud preview app deploy app.yaml --promote
 
 [google/ruby]: https://registry.hub.docker.com/u/google/ruby-runtime/
```
Comment (JustinBeckwith) [inline README.md]: (for other branches too) This actually isn't accurate any more :)  We don't use the docker hub image, we push them to gcr.io.  Could we just include a link to the ruby-docker github repo?

github.com/GoogleCloudPlatform/ruby-docker

File: app.yaml [hunk 2]
```diff
@@ -12,10 +12,11 @@
 # limitations under the License.
 
 # [START runtime]
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
```
Comment (JustinBeckwith) [inline app.yaml]: drop api_version: and module:

File: README.md (hunk 1 above)
Comment (jeffmendoza) [inline README.md]: Oh, yep, actually I was removing references to this link, but need to remove the actual link.

File: app.yaml (hunk 2 above)
Comment (jeffmendoza) [inline app.yaml]: will do

**Comment by JustinBeckwith** [comment]
LGTM

**Pull request merged**